pkg/awsutils/awsutils.go

```diff
@@ -662,15 +673,29 @@ func (cache *EC2InstanceMetadataCache) tagENI(eniID string) {
 //containsAttachmentLimitExceededError returns whether exceeds instance's ENI limit
 func containsAttachmentLimitExceededError(err error) bool {
 	if aerr, ok := err.(awserr.Error); ok {
-		return aerr.Code() == "AttachmentLimitExceeded"
+		return aerr.Code() == errAttachmentLimitExceeded
 	}
 	return false
 }
 
 // containsPrivateIPAddressLimitExceededError returns whether exceeds eni's IP address limit
 func containsPrivateIPAddressLimitExceededError(err error) bool {
 	if aerr, ok := err.(awserr.Error); ok {
-		return aerr.Code() == "PrivateIpAddressLimitExceeded"
+		return aerr.Code() == errPrivateIpAddressLimitExceeded
```

Comment:
should prob use this to get the original error type https://godoc.org/github.com/pkg/errors#hdr-Retrieving_the_cause_of_an_error